### PR TITLE
ref(general): Tiny fixes

### PR DESCRIPTION
**Pull request by lobsterkatie**
Just some stuff I came across while working on a bigger change ~, plus some prettier-induced changes~.

EDIT: Prettier changes pulled out into separate PR.

### CODE REVIEW

File: src/wizard/javascript/index.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ Sentry.init({
 
 The above configuration captures both error and performance data. To reduce the volume of performance data captured, change `tracesSampleRate` to a value between 0 and 1.
 
-One way to verify your setup is by intentionally sending an event that breaks your application.
+One way to verify your setup is by intentionally causing an error that breaks your application.
```
Comment (rhcarvalho) [inline src/wizard/javascript/index.md:35]: Check this sentence may appear in other places, I have a vague remembrance of that.

**Review by rhcarvalho — APPROVED** [review]
Very good :)

File: src/wizard/javascript/index.md (hunk 1 above)
Comment (lobsterkatie) [reply src/wizard/javascript/index.md:35]: Holy cow, you weren't kidding.

Fixed the rest and also ran prettier, because why not?

**Comment by dashed** [comment]
Changes to the Discover and Performance docs are fine to me 👍

**Comment by lobsterkatie** [comment]
> Changes to the Discover and Performance docs are fine to me 👍

LOL - that's because the only change to your stuff was removing an apostrophe in "it's." 😛

File: src/docs/clients/cocoa/migration.mdx [hunk 2]
```diff
@@ -13,23 +13,23 @@ tags: []
...
+source 'https://github.com/CocoaPods/Specs.git'
+platform :ios, '8.0'
+use_frameworks!
 
-  target 'YourApp' do
-      pod 'Sentry', :git => 'https://github.com/getsentry/sentry-cocoa.git', :subspecs => ['Core', 'KSCrash'], :tag => '3.x.x'
-  end
-  ```
+target 'YourApp' do
+    pod 'Sentry', :git => 'https://github.com/getsentry/sentry-cocoa.git', :subspecs => ['Core', 'KSCrash'], :tag => '3.x.x'
```
Comment (rhcarvalho) [inline src/docs/clients/cocoa/migration.mdx:22]: Surprised that prettier kept 4 spaces indentation here, Ruby code typically uses 2 spaces.

**Review by rhcarvalho — APPROVED** [review]
With 52 files changed, this is not-so-tiny anymore. Merge before you start getting conflicts ;)

File: src/docs/clients/cocoa/migration.mdx (hunk 2 above)
Comment (lobsterkatie) [reply src/docs/clients/cocoa/migration.mdx:22]: Yeah, huh. Dunno. Will fix.

**Comment by lobsterkatie** [comment]
> With 52 files changed, this is not-so-tiny anymore. Merge before you start getting conflicts ;)

Fair point. Just drunk with auto-formatting power, I guess! 🙂 

Have pulled the prettier changes into https://github.com/getsentry/sentry-docs/pull/2267 and will rebase this on that as soon as it clears.

**Review by dashed — APPROVED** [review]

**Pull request merged**